### PR TITLE
Integrate warmer into GCE jobs infrastructures

### PR DESCRIPTION
**Pull request by meatballhat**

### CODE REVIEW

File: bin/generate-latest-gce-images [hunk 1]
```diff
@@ -4,7 +4,7 @@
 require 'json'
 
 DEFAULT_GCE_PROJECT_ID = 'eco-emissary-99515'
-DEFAULT_IMAGE_PREFIXES = 'bastion nat tfw'
+DEFAULT_IMAGE_PREFIXES = 'bastion'
```
Comment (Lyoness) [inline bin/generate-latest-gce-images]: I assume this is a change unrelated to to warmer and just changes made to NAT or `tfw`

File: gce-production-1/main.tf [hunk 2]
```diff
@@ -170,9 +174,23 @@ resource "google_project_iam_member" "staging_1_workers" {
...
+}
+
 resource "google_project_iam_member" "production_2_workers" {
   count   = "${length(data.terraform_remote_state.production_2.workers_service_account_emails)}"
   project = "${var.project}"
   role    = "roles/compute.imageUser"
   member  = "serviceAccount:${element(data.terraform_remote_state.production_2.workers_service_account_emails, count.index)}"
 }
+
+resource "google_project_iam_member" "production_2_warmer" {
```
Comment (Lyoness) [inline gce-production-1/main.tf]: only other nit: why we name this member (and production workers) with `2` while staging  has`1`, given that we are in `gce-production-1`s main.tf?  Any significance?

**Review by Lyoness — APPROVED** [review]
Just curious questions in review.  LGTM!

File: bin/generate-latest-gce-images (hunk 1 above)
Comment (meatballhat) [reply bin/generate-latest-gce-images]: I was getting an image lookup failure and realized that this list was longer than necessary :man_shrugging:

File: gce-production-1/main.tf (hunk 2 above)
Comment (meatballhat) [reply gce-production-1/main.tf]: This resource only exists in `gce-production-1` and the purpose it to grant cloud images usage access to the _other_ `gce-.+` projects, which is why this resource has this name.

**Comment by 966556296635** [comment]
> _No description provided._

**Pull request merged**